parquet/src/main/java/org/apache/iceberg/parquet/ParquetFilters.java

```diff
@@ -168,9 +169,9 @@ protected Expression bind(UnboundPredicate<?> pred) {
     }
   }
 
-  private static
-  <C extends Comparable<C>, COL extends Operators.Column<C> & Operators.SupportsLtGt>
-  FilterPredicate pred(Operation op, COL col, C value) {
+  @SuppressWarnings("checkstyle:MethodTypeParameterName")
+  private static <C extends Comparable<C>, COL extends Operators.Column<C> & Operators.SupportsLtGt> FilterPredicate
+      pred(Operation op, COL col, C value) {
```

Comment:
`FilterPredicate` can't be on the new line? I think I liked the formatting better before, but it's fine if there's a checkstyle problem with it.

Reply:
I think it was the indentation, placed `FilterPredicate` on the new line again.